### PR TITLE
Display following followers

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
OK

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,6 +10,13 @@ def show
     @user_books = UserBook.by_user(current_user.id).page(params[:page]).per 5
     @books = Book.favourite(current_user.id).page(params[:fbook_page]).per 5
     @user_activities = User.review_activities current_user
+    if params[:following]
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: why not use meta programming at here?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -10,6 +10,8 @@ def show
     @user_books = UserBook.by_user(current_user.id).page(params[:page]).per 5
     @books = Book.favourite(current_user.id).page(params[:fbook_page]).per 5
     @user_activities = User.review_activities current_user
+
+    load_follow_dependencies(params[:following] ? :following : :followers)
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: I don't understand it, may be wrong codition

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,19 +1,40 @@
-<h1><%= t :user_profile %></h1>
-<div class="row">
-  <div class="col-md-5">
-    <p><%= render "user_picture" %></p>
-    <%= render "shared/stats" %>
-    <%= render "follow_form" %><br>
-    <p><b><%= t :name %>: <%= @user.name %></b></p>
-    <p><%= t :email %>: <%= @user.email %></p>
+<% if params[:followers] || params[:following] %>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: why not `params[:type]`, it is followers or following

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (shakil-fg) [inline app/controllers/users_controller.rb]: It's working correctly. the block
  params[:following] ? :following : :followers
returning :following or :follower (based on the input parameter from _stats.html.erb)

**Pull request merged**